src/useSprings.js

```diff
@@ -24,11 +24,14 @@ export const useSprings = (length, props) => {
         return c.update(newProps)
       }),
       // This updates the controllers with new props
-      props =>
+      props => {
+        const isFn = is.fun(props)
+        if (!isFn) props = toArray(props)
```

Comment:
Previously, this function would only take an array or a function. Now you can pass an object without `Controller#update` receiving an undefined value on accident.

This affected `useSpring` in particular.